src/scene/materials/standard-material-options-builder.js

```diff
@@ -347,8 +351,17 @@ class StandardMaterialOptionsBuilder {
                 if (stdMat[uname] === 0 && !hasUv0) allow = false;
                 if (stdMat[uname] === 1 && !hasUv1) allow = false;
                 if (allow) {
+
```

Comment:
you mentioned you wanted to add some comments .. please do so, next year it will be less clear